packages/next-on-pages/tests/templates/requestTestData/configRewritesRedirectsHeaders.ts

```diff
@@ -60,7 +60,7 @@ const rawVercelConfig: VercelConfig = {
 		{ handle: 'resource' },
 		{
 			src: '^(?:/((?:[^/]+?)(?:/(?:[^/]+?))*))?(?:/)?$',
-			dest: 'https://my-old-site.com/$1',
+			dest: 'https://uuid.rocks/$1',
```

Comment:
as you mentioned in the PR's description 😅 ...

I really don't like us relying to some external site for the tests (and also the tests should ideally be able to run locally without an internet connection I think)

so I'd prefer it if we could mock the fetch instead of it actually happening, something along these lines:
![Screenshot 2023-07-24 at 12 35 14](https://github.com/cloudflare/next-on-pages/assets/61631103/36618515-1a0e-4da1-8380-15ea65b3676f)

What do you think? does this look ok to you? 🙂

Reply:
i thought you might have been a bit unhappy about mocking fetch ngl, but if you are happy with it i am happy to do that